esp/scm/additional.cmake

```diff
@@ -39,11 +39,11 @@ foreach ( loop_var ${ESPSCM_SRCS} )
     if (SCM_BUILD)
       add_custom_command ( DEPENDS hidl ${ESPSCM_SOURCE_DIR}/${loop_var}
                            OUTPUT ${ESPSCM_GENERATED_DIR}/${result}.esp ${ESPSCM_GENERATED_DIR}/${result}.hpp ${ESPSCM_GENERATED_DIR}/${result}.int ${ESPSCM_GENERATED_DIR}/${result}.ipp ${ESPSCM_GENERATED_DIR}/${result}_esp.cpp ${ESPSCM_GENERATED_DIR}/${result}_esp.ipp
-                           COMMAND $<TARGET_FILE:hidl> ${ESPSCM_SOURCE_DIR}/${result}.ecm ${ESPSCM_GENERATED_DIR}
+                           COMMAND ASAN_OPTIONS=detect_leaks=0 $<TARGET_FILE:hidl> ${ESPSCM_SOURCE_DIR}/${result}.ecm ${ESPSCM_GENERATED_DIR}
```

Comment:
NB: I have opened https://track.hpccsystems.com/browse/HPCC-28588 to pursue the leaks caused by hidl and esdl-xml, but those fixes are non-trivial and can be dealt with separately.